presto-main/src/main/java/com/facebook/presto/operator/scalar/JoniRegexpFunctions.java

```diff
@@ -307,6 +309,16 @@ public static Block regexpSplit(@SqlType("varchar(x)") Slice source, @SqlType(Jo
         return blockBuilder.build();
     }
 
+    private static int getMatchingOffset(Matcher matcher, int at, int range, int option)
+    {
+        try {
```

Comment:
In my testing I noticed this. Would be helpful for this piece of knowledge to be available to anyone reading the code.
```suggestion
        try {
            // Joni is not guaranteed to detect that interrupt was set with simple expressions
```